@dcrowe I faced an issue when server response has  `diffName` as `None` 
added test case for it, but I'm not able to fix it :(

```
Traceback (most recent call last):
  File "example.py", line 21, in <module>
    vrt.track(TestRun(
  File "/Users/pashidlos/work/vrt/sdk-python/visual_regression_tracker/visualRegressionTracker.py", line 91, in track
    result = self._submitTestResult(test)
  File "/Users/pashidlos/work/vrt/sdk-python/visual_regression_tracker/visualRegressionTracker.py", line 86, in _submitTestResult
    testRunResult = _from_dict(result, TestRunResponse)
  File "/Users/pashidlos/work/vrt/sdk-python/visual_regression_tracker/types.py", line 91, in _from_dict
    obj = dacite.from_dict(clazz, data)
  File "/Users/pashidlos/.conda/envs/sdk-python/lib/python3.8/site-packages/dacite/core.py", line 65, in from_dict
    raise WrongTypeError(field_path=field.name, field_type=field.type, value=value)
dacite.exceptions.WrongTypeError: wrong value type for field "diffName" - should be "str" instead of value "None" of type "NoneType"
```

--- a/visual_regression_tracker/types.py
+++ b/visual_regression_tracker/types.py
@@ -49,14 +49,14 @@ class TestRunStatus(enum.Enum):
 class TestRunResponse:
     id: str = None
     imageName: str = None
-    diffName: str = None
-    baselineName: str = None
+    diffName: typing.Optional[str] = None
+    baselineName: typing.Optional[str] = None
     url: str = None
     merge: bool = False
     status: TestRunStatus = None
-    pixelMisMatchCount: float = None
-    diffPercent: float = None
-    diffTollerancePercent: float = None
+    pixelMisMatchCount: typing.Optional[float] = None
+    diffPercent: typing.Optional[float] = None
+    diffTollerancePercent: typing.Optional[float] = None
 
 
 @dataclasses.dataclass

